mne/defaults.py

```diff
@@ -33,7 +34,7 @@
     titles=dict(mag='Magnetometers', grad='Gradiometers', eeg='EEG', eog='EOG',
                 ecg='ECG', emg='EMG', misc='misc', seeg='sEEG', bio='BIO',
                 dipole='Dipole', ecog='ECoG', hbo='Oxyhemoglobin',
-                ref_meg='Reference Magnetometers',
+                ref_meg='Reference Magnetometers', fnirs_raw='fNIRS',
```

Comment:
These will need `fnirs_od` eventually I think but we can add that when you add `od`